### PR TITLE
Telemetry fix

### LINKED ISSUE #1360: Telemetry missing key properties - Telemetry Initializer properties missing?
## Project
**Name**: Virtual Assistant
**Language**: C#

## Description
The telemetry produced by the Virtual Assistant doesn't contain important properties as contained in the documentation.

It appears the properties that the Telemetry Initializer are supposed to create are missing.

The session_id and activityId are critical to being able to properly analyze complete conversations.

## To Reproduce
Build VirtualAssistantSample
Run
Type in something
Look at telemetry.

## Expected behavior
Luis Documentation:
LuisTelemetry

Actual Luis Telemetry:
ActualLuisTelemetry

QnA Documentation:
QnATelemetry

Actual QnA Telemetry:
ActualQnATelemetry

Note: the question and matchedQuestion properties are perfect.  Please don't make them match the documentation.  Fix the documentation.

### PR DESCRIPTION
**Pull request by munozemilio**
Fixes #1360

### CODE REVIEW

File: templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json [hunk 1]
```diff
@@ -7,9 +7,8 @@
       "provider": ""
     }
   ],
-  "appInsights": {
-    "appId": "",
-    "instrumentationKey": ""
+  "ApplicationInsights": {
+    "InstrumentationKey": ""
```
Comment (lzc850612) [inline templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json:11]: is it necessary to remove appId in here? this change will break our deployment script as it'll try to fill the appId field and it doesn't exist here any more.

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json (hunk 1 above)
Comment (munozemilio) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json:11]: You can keep both but the telemetry client has to be set as per [documentation](https://docs.microsoft.com/en-gb/azure/azure-monitor/app/asp-net-core).

Comment (darrenj) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json:11]: Indeed - we seem to have diverged somewhere along the line from the standard AppInsights telemetry key.

Comment (lzc850612) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json:11]: yeah i was mistaken in my original comment. though we just removed the 'appid' but rather we added ApplicationInsights and removed appInsights.

we still have to make proper deployment script change though

Comment (ryanisgrig) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json:11]: nit: does the casing of `ApplicationInsights`/`InstrumentationKey` matter? If not we should use camel case.

Comment (munozemilio) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/appsettings.json:11]: I've never tried but should be simple to test.

**Comment by lauren-mills** [comment]
@munozemilio I pulled these changes in to this PR https://github.com/microsoft/botframework-solutions/pull/1560

**Pull request closed without merging**